lxc/storage_volume.go

```diff
@@ -1391,16 +1391,18 @@ type cmdStorageVolumeList struct {
 	flagColumns     string
 	flagAllProjects bool
 
+	allVolumes bool
```

Comment:
I'm not sure this needs to be defined on the command struct itself. Is this so that it is accessible in `parseColumns`? If so, there is already a `clustered` boolean argument, so we could perhaps pass `allVolumes` in as an argument as well.